crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -403,7 +403,7 @@ define_tedge_config! {
             include: {
                 /// Set the bridge local clean session flag (this requires mosquitto >= 2.0.0)
                 #[tedge_config(note = "If set to 'auto', this cleans the local session accordingly the detected version of mosquitto.")]
-                #[tedge_config(example = "auto", default(value = false))]
+                #[tedge_config(example = "auto", default(value = "auto"))]
```

Comment:
@jarhodes-sag  I was surprised to be unable to set `default(value = AutoFlag::Auto)` here as this leads to an error: Unexpected literal type `path`. Same thing with  `default(value = Auto)` .

Not a big deal as there is a workaround, but surprising.

Reply:
> Ah, for that you'd want `default(variable = "AutoFlag::Auto")`

Thanks. I will use that then.

>  the name "variable" is a bit confusing here, but I couldn't think of a better term

That's okay. The issue is more being aware of these subtleties.

Reply:
To improve the awareness aspect, I've added a custom error message: https://github.com/thin-edge/thin-edge.io/pull/2401

Reply:
Thanks